styles/main.css

```diff
@@ -2,3 +2,97 @@
 :root {
   --bs-font-sans-serif: 'Roboto', system-ui, -apple-system, "Segoe UI", "Helvetica Neue", Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
 }
+
+
+div {
+  display: block;
+}
+
+/********************/
+/*    Customers     */
+/********************/
+
+.blue {
+  color: #00cec9;
+}
+.img-fluid {
+  max-width: 100%;
+  height: auto;
+  margin-left: auto;
+  display: block;
+}
```

Comment:
Bootstrap [already has this](https://getbootstrap.com/docs/5.0/content/images/#responsive-images).

```suggestion
```

Reply:
Reviewed and corrected